### DISCUSSION TITLE
Proposal: rename ‘ownership-or-control statement’ to ‘relationship statement’

### DISCUSSION THREAD

**Discussion by kd-ods** [openownership/data-standard#474] [Announcements]
We have been using the term 'ownership-or-control statement’ to refer to the object that contains information about ownership interests or control interests.

I propose renaming it to 'relationship statement'. The benefits are:

- It's less clunky
- It highlights the object's nature as a connection between a subject and an interested party.
- It reinforces the graph/network nature of BODS format.

**Comment by lgs85** [comment]
Thanks for this @kd-ods 

>It highlights the object's nature as a connection between a subject and an interested party.

In my experience misunderstanding how ownership or control statements work has been the main stumbling block for implementers quickly getting to grips with the BODS data model. I think that renaming OOC statements to relationship statements will have a clear practical effect in helping users understand BODS and its structure.

**Comment by kd-ods** [comment]
This proposal has not raised any questions or prompted concerns. We will be adopting this update of terminology from the next BODS version, v0.4.
